spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -326,7 +326,7 @@ def aggregate_per_slice_or_level(metric, mask=None, slices=[], levels=[], distan
                 else:
                     mask_slicegroup = np.ones(data_slicegroup.shape)
                 # Ignore nonfinite values
-                i_nonfinite = np.where(np.isfinite(data_slicegroup) == False)
+                i_nonfinite = np.where(np.isfinite(data_slicegroup) == False)  # noqa: E712
```

Comment:
This seems reasonable, as per https://github.com/PyCQA/pycodestyle/issues/450#issuecomment-390169995!